pyanaconda/modules/payload/live/live_os.py

```diff
@@ -48,12 +45,31 @@ class LiveOSHandlerModule(PayloadHandlerBase):
     def __init__(self):
         super().__init__()
 
-        self._image_path = ""
-        self.image_path_changed = Signal()
-
         self._kernel_version_list = []
         self.kernel_version_list_changed = Signal()
 
+    @property
+    def supported_source_kinds(self):
+        """Get list of supported sources by Live Image module."""
```

Comment:
"... list of sources supported by Live OS module."

Reply:
Fixed.